README.md

```diff
@@ -38,6 +38,27 @@ To run the tests, you also must have `nose` installed.
 
 ### User Installation
 
+#### Fedora Release 25 onwards
+
+Install Pomegranate and its dependencies ( for Python3 )
+
+```
+$ sudo dnf install -y python3-virtualenv python3-scipy python3-numpy python3-networkx python3-Cython
+$ virtualenv env3 --python=python3.5 --system-site-packages
```

Comment:
Is it possible to do this without a virtualenv? It seems like it might throw off some newer users by adding the additional complexity of virtualenvs to the installation process.